File: src/shared/dishes/dish.go
package dishes

const (
	Soup = 0
	Garnish = 1
	Main = 2
	Salad = 3
)


type Dish struct {
	Id int `db:"id" json:"id"`
	Label string `db:"label" json:"label" validate:"required"`
	Description string `db:"description" json:"description,omitempty"`
	Type int `db:"type" json:"type"`
	PhotoUrl string `db:"photo_url" json:"photoUrl"`
	HasGarnish bool `db:"has_garnish" json:"hasGarnish"`
}

File: src/ui/src/app/shared/interfaces/dish.ts
export enum DishType {
  soup,
  garnish,
  main,
  salad,
  special
}

export const DISH_TYPES = [
  'Soup',
  'Garnish',
  'Main',
  'Salad',
  'Special'
];

export const DISH_TYPE_COLORS = [
  'purple',
  'orange',
  'info',
  'success',
  'danger'
];

export const DISH_IMG_DEFAULT = '/assets/dish_no_image.jpg';

export interface IDish {
  id?: number;
  label: string;
  description: string;
  type: number;
  photoUrl: string;
}
